### ISSUE TITLE
New Hadoop JMX plugin

### ISSUE THREAD

**Issue by harisekhon** [harisekhon/nagios-plugins#80]
@david-caldwell

Following on from #78 splitting the HBase metrics issue raised there...

This may have changed in newer versions of Hadoop. I've tested it against HBase 0.98 on HDP 2.2 and the check_hbase_metrics.pl plugin gives
```UNKNOWN: no stats collected from /metrics page (daemon recently started?)```
as expected since HBase is returning a blank JSON document.

I've had a check_hadoop_jmx.pl plugin in the works for ages so I'll release a version of it soon, this evening after work hopefully. The newer plugin hits the alternate /jmx url you are referring to.

**Comment by harisekhon** [comment]
@david-caldwell

I've just uploaded ```check_hadoop_jmx.pl```... try that as a replacement for the check_hadoop_metrics.pl. I've had this in my working directory for a couple years, but you've just prompted me to finish it. I've also added a note to check_hadoop_metrics.pl to see this newer plugin.

Run the following as there are some library updates it requires that it will pull in:

```make update2```

and then see

```./check_hadoop_jmx.pl --help```

I've also added the following symlinks  which allow use of more specific environment variables as shown in ```--help``` and auto-default to the right ports:
```
check_hbase_master_jmx.pl
check_hbase_regionserver_jmx.pl
check_hadoop_namenode_jmx.pl
check_hadoop_datanode_jmx.pl
```
